network/test/middleware_test.go

```diff
@@ -1,12 +1,16 @@
 package test
 
 import (
+	"bytes"
 	"context"
 	"fmt"
+	"regexp"
 	"sync"
 	"testing"
 	"time"
 
+	"github.com/onflow/flow-go/network/p2p/middleware"
```

Comment:
nit: group import with the rest of the flow imports

Reply:
https://github.com/onflow/flow-go/pull/3675/commits/c91c88ec46b9cf835ac81dacb279b90cbfb397bd